php8.2-dev/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM thebiggive/php:8.2
```

Comment:
this file is a copy of https://github.com/thebiggive/docker-php/blob/c35993238ae6f62c3e3af70045a431d506ef08d7/dev/Dockerfile , just changed 8.1 to 8.2